src/tests/integration/IntegrationBase.t.sol

```diff
@@ -0,0 +1,938 @@
+        uint256 pbal = getLPTokenBalanceInAssets(address(pool));
+        assertGt(pgem, 0);
+        assertGt(pbal, 0);
+
+        address receiver = address(123);
+
+        vm.prank(admin); pool.quit(address(receiver));
+        vm.prank(admin); vat.slip(ilk, address(pool), -int256(pgem));
+
+        uint256 ngem = vat.gem(ilk, address(pool));
+        uint256 nbal = getLPTokenBalanceInAssets(address(pool));
+        assertEq(ngem, 0);
+        assertEq(nbal, 0);
+
+        uint256 gem = vat.gem(ilk, receiver);
+        uint256 bal = getLPTokenBalanceInAssets(receiver);
+        assertEq(gem, 0);
+        assertEq(bal, pbal);
+    }
+
+    function test_direct_deposit_mom() public {
+        adjustDebt(standardDebtSize);
+
+        (uint256 ink, ) = vat.urns(ilk, address(pool));
+        assertGt(ink, 0);
+        assertEq(plan.active(), true);
+
+        // Something bad happens - we need to bypass gov delay
+        vm.prank(admin); mom.disable(address(plan));
+
+        assertEq(plan.active(), false);
+
+        // Close out our position
+        hub.exec(ilk);
+
+        (ink, ) = vat.urns(ilk, address(pool));
+        assertRoundingEq(ink, 0);
+    }
+
+    function test_set_tau_not_caged() public {
+        (, , uint256 tau, , ) = hub.ilks(ilk);
+        assertEq(tau, 7 days);
+        vm.prank(admin); hub.file(ilk, "tau", 1 days);
+        (, , tau, , ) = hub.ilks(ilk);
+        assertEq(tau, 1 days);
+    }
+
+    function test_fully_unwind_debt_paid_back() public {
+        uint256 liquidityBalanceInitial = getLiquidity();
+
+        adjustDebt(standardDebtSize);
+
+        (uint256 pink, uint256 part) = vat.urns(ilk, address(pool));
+        uint256 gemBefore = vat.gem(ilk, address(pool));
+        uint256 viceBefore = vat.vice();
+        uint256 sinBefore = vat.sin(address(vow));
+        uint256 vowDaiBefore = vat.dai(address(vow));
+        uint256 liquidityBalanceBefore = getLiquidity();
+        uint256 assetsBalanceBefore = getLPTokenBalanceInAssets(address(pool));
+
+        // Someone pays back our debt
+        dai.setBalance(address(this), 10 * WAD);
+        dai.approve(address(daiJoin), type(uint256).max);
+        daiJoin.join(address(this), 10 * WAD);
+        vat.frob(
+            ilk,
+            address(pool),
+            address(pool),
+            address(this),
+            0,
+            -int256(10 * WAD)
+        );
+
+        (uint256 ink, uint256 art) = vat.urns(ilk, address(pool));
+        assertEq(ink, pink);
+        assertEq(art, part - 10 * WAD);
+        assertEq(ink - art, 10 * WAD);
+        assertEq(vat.gem(ilk, address(pool)), gemBefore);
+        assertEq(vat.vice(), viceBefore);
+        assertEq(vat.sin(address(vow)), sinBefore);
+        assertEq(vat.dai(address(vow)), vowDaiBefore);
+        assertRoundingEq(getLiquidity(), liquidityBalanceBefore);
+        assertApproxEqAbs(getLPTokenBalanceInAssets(address(pool)), assetsBalanceBefore, 2 * roundingTolerance); // rounding may affect twice
```

Comment:
```suggestion
        assertEq(getLiquidity(), liquidityBalanceBefore);
        assertEq(getLPTokenBalanceInAssets(address(pool)), assetsBalanceBefore, 2 * roundingTolerance);
```
Nothing was executed  that could actually change these values to have to round anything.